### PR TITLE
Spelling and formatting segments

### PR DESCRIPTION
**Pull request by wlach**
There were a few issues with the spelling and formatting with the recently landed segments changes, which this PR addresses.

### CODE REVIEW

File: src/concepts/segments.md [hunk 1]
```diff
@@ -105,7 +104,7 @@ So if a user was active as a regular user v3 on e.g. the second day of a 28-day
 
 Since many user states use the full extent of the `*_bits` column wizardry in `clients_last_seen`, you'll have to query WAU or MAU the old fashioned way:
 
-```lang=sql
+```sql
```
Comment (wlach) [inline src/concepts/segments.md:107]: Note that SQL code blocks need to be declared this way in order for the syntax highlighter to pick them up.

**Review by godelstheory — APPROVED** [review]
Thanks for doing this wlach, and sorry for the trouble! I will be mindful next time to get up to speed correctly applying `link_check.sh` and `prettier_fix.sh`.

**Comment by wlach** [comment]
> Thanks for doing this wlach, and sorry for the trouble! I will be mindful next time to get up to speed correctly applying `link_check.sh` and `prettier_fix.sh`.

Np! FWIW I consider the prettier formatting "optional" (it's trivial to reformat the repository), but it's good practice to make sure that spelling and link checking are passing (at least with respect to any changes you're making)

File: src/concepts/segments.md (hunk 1 above)
Comment (limpaopao) [reply src/concepts/segments.md:107]: > Note that SQL code blocks need to be declared this way in order for the syntax highlighter to pick them up.

Comment (limpaopao) [reply src/concepts/segments.md:107]: > > Note that SQL code blocks need to be declared this way in order for the syntax highlighter to pick them up.
> 
>

**Pull request merged**